app/gcl/associate-cloud-engineer/cloud-load-balancing-guide/page.module.css

```
@@ -0,0 +1,720 @@
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 3px;
+    width: 0%;
+    background: linear-gradient(90deg, var(--color-theme-ace-accent), var(--color-google-blue));
+    z-index: 100;
+    transition: width 0.1s linear;
+}
+
+/* ── レイアウト (PC優先の flex / sticky / max-width 制御) ── */
+.container :global(.layout) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)